src/Toolkit/Toolkit.UI.Controls/MeasureToolbar/MeasureToolbar.cs

```diff
@@ -112,10 +113,14 @@ public MeasureToolbar()
                     Geometry.AreaUnits.SquareMillimeters,
                     Geometry.AreaUnits.SquareYards,
                 };
-            LineSketchEditor = new SketchEditor();
-            AreaSketchEditor = new SketchEditor();
-            SelectionLineSymbol = LineSketchEditor.Style?.LineSymbol;
-            SelectionFillSymbol = AreaSketchEditor.Style?.FillSymbol;
+            LineGeometryEditor = new GeometryEditor();
+            AreaGeometryEditor = new GeometryEditor();
+            SelectionLineSymbol = LineGeometryEditor.Tool?.Style?.LineSymbol;
+            SelectionFillSymbol = AreaGeometryEditor.Tool?.Style?.FillSymbol;
```

Comment:
What happens if the tool or style is null? Then the symbol wouldn't get set. If you know they are always set, resolve the warning with `!` and comment what the default is or explicitly instantiate it. 
Furthermore, the geometry editor is public here, so the developer could be modifying the editors and remove/replace/reconfigure the tool.
Does it make more sense to not expose the various geometry editors instances, just have one editor internally, and instead expose the various tool configurations?